Dockerfile

```diff
@@ -1,41 +1,28 @@
-FROM centos:6
+FROM ruby:2.3-slim-jessie
 
 ARG sidekiq_license
 ENV BUNDLE_ENTERPRISE__CONTRIBSYS__COM=$sidekiq_license
 ARG exclude_sidekiq_ent
 ENV EXCLUDE_SIDEKIQ_ENTERPRISE=$exclude_sidekiq_ent
+ENV APP_PATH /src/vets-api
 
 # Match the jenkins uid/gid on the host (504)
 RUN groupadd -r vets-api && \
       useradd -r -g vets-api vets-api
 
-RUN yum install -y git make gcc-c++ openssl-devel readline-devel zlib-devel sqlite-devel postgresql-devel socat timeout epel-release nc
+RUN apt-get update -qq && apt-get install -y build-essential \
+		                             git \
+                                             libpq-dev \
+                                             nodejs \
```

Comment:
nodejs is a new package unless there was something on the old system that had it as a dependency.

Reply:
👍 yep you're right, think this was left over from one of my initial tests